symbols/overlay29.yml

```diff
@@ -6797,28 +6940,41 @@ overlay29:
         r1: item pointer
         r2: some flag?
         return: success flag
-    - name: SpawnEnemyItemDropWrapper
+    - name: RemoveGroundItem
+      address:
+        EU: 0x23462A8
+        NA: 0x23456BC
+      description: |-
+        Removes an item lying on the ground.
+        
+        Also updates dungeon::n_items.
+        
+        r0: Position where the item is located
+        r1: If true, update dungeon::poke_buy_kecleon_shop and dungeon::poke_sold_kecleon_shop
+    - name: PlaceThrownItemWrapper
       address:
         EU: 0x2346628
         NA: 0x2345A3C
         JP: 0x2346E00
       description: |-
-        Wraps SpawnEnemyItemDrop in a more convenient interface.
+        Wraps PlaceThrownItem in a more convenient interface.
         
         r0: entity
         r1: position
         r2: item
         r3: ?
-    - name: SpawnEnemyItemDrop
+    - name: PlaceThrownItem
```

Comment:
Seems to need renaming per discussion

Reply:
Renamed to `SpawnDroppedItem` as suggested.

Reply:
I tweaked the description a bit more, but otherwise this looks good.